include/picongpu/plugins/openPMD/restart/LoadSpecies.hpp

```diff
@@ -143,7 +143,9 @@ namespace picongpu
                     mallocMem;
                 mallocMem(buffers, mappedFrame, maxChunkSize);
 
-                uint32_t const numLoadIterations = alpaka::core::divCeil(totalNumParticles, maxChunkSize);
+                uint32_t numLoadIterations = 0u;
```

Comment:
```suggestion
                uint32_t numLoadIterations = (maxChunkSize == 0u) ? 0u : alpaka::core::divCeil(totalNumParticles, maxChunkSize);
```
Looks fine, as a matter of style this is maybe cleaner. But completely up to you